tests/api_tests/test_participant_summary_api.py

```diff
@@ -465,6 +465,77 @@ def test_check_login(self):
         self.assertEqual(null_email_result.json['message'],
                          'Missing email or login_phone_number in request')
 
+    def test_invalid_filters_return(self):
+        num_summary = 10
+        first_name = "Testy"
+        for num in range(num_summary):
+            if num == 1:
+                self.data_generator \
+                    .create_database_participant_summary(
+                        firstName=first_name,
+                        lastName="Tester"
+                    )
+            else:
+                self.data_generator \
+                    .create_database_participant_summary()
+
+        response_good_bad_filter = self.send_get(f"ParticipantSummary?foobarbaz=1&firstName={first_name}")
+        self.assertEqual(len(response_good_bad_filter['entry']), 1)
+        resource = response_good_bad_filter['entry'][0]['resource']
+        self.assertEqual(resource['firstName'], 'Testy')
+        self.assertEqual(resource['lastName'], 'Tester')
+
+        response_good_filter = self.send_get(f"ParticipantSummary?firstName={first_name}")
+        self.assertEqual(len(response_good_filter['entry']), 1)
+        resource = response_good_filter['entry'][0]['resource']
+        self.assertEqual(resource['firstName'], 'Testy')
+        self.assertEqual(resource['lastName'], 'Tester')
+
+        response_bad_filter = self.send_get(
+            "ParticipantSummary?foobarbaz=1",
+            expected_status=http.client.BAD_REQUEST
+        )
+        self.assertEqual(response_bad_filter.status_code, 400)
+        self.assertEqual(response_bad_filter.json['message'], 'No valid fields were provided')
+
+        response_no_filter = self.send_get("ParticipantSummary")
+        self.assertEqual(len(response_no_filter['entry']), num_summary)
+
+    def test_constraints_dob_and_lastname(self):
+        num_summary = 3
+        date_str = datetime.date(1978, 10, 9)
```

Comment:
nit: I might be using the wrong convention, but I would assume a variable holds a string if it ends with `_str` like this. But `date_str` holds a Date object. Should it be changed?